### PR TITLE
[FLINK-36276] Fix the description of MySqlDataSourceOptions.`scan.sta…

### PR DESCRIPTION
**Pull request by LinMingQiang**
…rtup.mode`.

### CODE REVIEW

File: flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSourceOptions.java [hunk 1]
```diff
@@ -126,8 +126,8 @@ public class MySqlDataSourceOptions {
                     .defaultValue("initial")
                     .withDescription(
                             "Optional startup mode for MySQL CDC consumer, valid enumerations are "
-                                    + "\"initial\", \"earliest-offset\", \"latest-offset\", \"timestamp\"\n"
-                                    + "or \"specific-offset\"");
+                                    + "\"initial\", \"earliest-offset\", \"latest-offset\", \"timestamp\", \"specific-offset\"\n,"
+                                    + " or \"snapshot\"");
```
Comment (yuxiqian) [inline flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSourceOptions.java:130]: Seems `\n` here isn't necessary

Comment (LinMingQiang) [reply flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSourceOptions.java]: resolved

**Review by yuxiqian — APPROVED** [review]
Thanks for @LinMingQiang's contribution!

**Comment by LinMingQiang** [comment]
@yuxiqian  Can you help me merge this PR?

**Comment by yuxiqian** [comment]
Could @ruanhang1993 please take a look?

**Pull request closed without merging**